### PR TITLE
[INLONG-5459][Sort] Add audit for MySQL extract node

### LINKED ISSUE #5459: [Feature][Sort] Add audit for MySQL extract node
### Description

Add Audit for mysql extract node

### Use case
User only need add with param "inlong.audit":"\<host\>&\<port\>" when using flink sql.
But user must set inlong.metric when set inlong.audit.
for example:
with param "inlong.metric":"\<groupId\>& \<streamId\> & \<nodeId\>" 

example:
```sql
  create table `table_groupId_streamId_nodeId1`(
    `age` INT,
    `name` STRING,
    PRIMARY KEY(`age`) NOT ENFORCED)
    WITH (
    'connector' = 'mysql-cdc-inlong',
    'hostname' = 'xxxx',
    'username' = 'xxx',
    'password' = 'xxx',
    'database-name' = 'test',
    'scan.incremental.snapshot.enabled' = 'true',
    'server-time-zone' = 'GMT+8',
    'table-name' = 'user',
   'inlong.audit' = 'host&port',
   'inlong.metric' = 'mysqlGroup&mysqlStream&mysqlNode1'
);

CREATE TABLE `table_groupId_streamId_nodeId2`(
    PRIMARY KEY (`name`,`age`) NOT ENFORCED,
    `name` STRING,
    `age` INT)
    WITH (
    'connector' = 'print'
);

   INSERT INTO `table_groupId_streamId_nodeId2`
    SELECT
    `name` AS `name`,
    `age` AS `age`
    FROM `table_groupId_streamId_nodeId1`;
```

It will send audit data to [audit service](https://github.com/apache/inlong/tree/master/inlong-audit)

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gong**
### Prepare a Pull Request

- [INLONG-5459][Sort] Add Audit for mysql extract node

- Fixes #5459 

### Motivation

Add Audit for mysql extract node

### Modifications

* Add inlong audit option for mysql cdc connector
* Add audit computing for mysql extract node

### CODE REVIEW

File: inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/debezium/DebeziumSourceFunction.java [hunk 1]
```diff
@@ -408,12 +417,17 @@ public void run(SourceContext<T> sourceContext) throws Exception {
...
         metricGroup.gauge(
                 "sourceIdleTime", (Gauge<Long>) () -> debeziumChangeFetcher.getIdleTime());
-        if (StringUtils.isNotEmpty(this.inlongMetric)) {
-            String[] inlongMetricArray = inlongMetric.split(Constants.DELIMITER);
-            String groupId = inlongMetricArray[0];
-            String streamId = inlongMetricArray[1];
-            String nodeId = inlongMetricArray[2];
-            sourceMetricData = new SourceMetricData(groupId, streamId, nodeId, metricGroup);
+        if (StringUtils.isNotEmpty(this.inLongMetric)) {
+            String[] inLongMetricArray = inLongMetric.split(Constants.DELIMITER);
```
Comment (healchow) [inline inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/debezium/DebeziumSourceFunction.java:421]: Please keep the naming of `inlongMetric` and `inlongMetricArray`, and change the `inLongAudit` to `inlongAudit`, thanks.

File: inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/util/ValidateMetricOptionUtils.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import org.apache.flink.table.api.ValidationException;
+
+/**
+ * validate option tool
+ */
+public class ValidateMetricOptionUtils {
+
+    /**
+     * validate inlong metric when set inlong audit
+     * @param inLongMetric
```
Comment (healchow) [inline inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/util/ValidateMetricOptionUtils.java:30]: 1. Use `inlongMetric`, `inlongAudit`.
2. Add a description for each param, thanks.

File: inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/source/metrics/MySqlSourceReaderMetrics.java [hunk 3]
```diff
@@ -59,6 +61,10 @@ public class MySqlSourceReaderMetrics {
     private static String STREAM_ID = "streamId";
     private static String GROUP_ID = "groupId";
     private static String NODE_ID = "nodeId";
+    private String inLongGroupId;
```
Comment (healchow) [inline inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/source/metrics/MySqlSourceReaderMetrics.java:64]: Please change to `inlongGroupId` and `inlongStreamId`.

**Review by healchow — CHANGES_REQUESTED** [review]

File: inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/util/ValidateMetricOptionUtils.java (hunk 2 above)
Comment (gong) [reply inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/util/ValidateMetricOptionUtils.java]: done

File: inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/table/MySqlTableSource.java [hunk 4]
```diff
@@ -182,7 +185,8 @@ public MySqlTableSource(
             Properties jdbcProperties,
             Duration heartbeatInterval,
             boolean migrateAll,
-            String inlongMetric) {
+            String inLongMetric,
```
Comment (healchow) [inline inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/table/MySqlTableSource.java:188]: Please change it to `inlongXxx`, thanks.

Comment (gong) [reply inlong-sort/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/table/MySqlTableSource.java]: done

**Review by EMsnap — APPROVED** [review]

**Review by Oneal65 — APPROVED** [review]

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**